application-xpoll-ui/src/main/resources/XPoll/XPollSheet.xml

```diff
@@ -172,27 +172,33 @@
               #end ##
             &lt;/tr&gt;
           #end
+
+          ## third table row
+          &lt;tr&gt;
+            &lt;td&gt;
+              $services.localization.render('contrib.xpoll.numberVotes')
+            &lt;/td&gt;
+            #foreach ($proposal in $proposals) &lt;td&gt; $!voteCount.get($proposal) &lt;/td&gt; #end
+          &lt;/tr&gt;
+
+          ## Separate the current user if he hasn't voted yet.
           #if (!$foundUser &amp;&amp; $xcontext.user != 'XWiki.XWikiGuest')
+            &lt;tr&gt;
+              &lt;td class="noBorder"&gt;$services.localization.render('contrib.xpoll.notVoted')&lt;/td&gt;
+              #foreach ($proposal in $proposals) &lt;td class="noBorder"&gt;&lt;/td&gt; #end
```

Comment:
There is no need to generate all the table cells to fill the entire table row. The visual result will the the same without this line.